Content/src/Client/client-minimal.fs

```diff
@@ -0,0 +1,73 @@
+module Client
```

Comment:
Extracting a separate file like that makes it impossible to work on the template for minimal version (it won't compile)

Reply:
Do you work on the template directly? Sorry - I thought that that wasn't even possible. We just worked off a separate repository, got it working and then copied the files over into here.

Reply:
It wasn't possible with v1, but it is possible with just the full and minimal options. I'll write a short instruction on how to do that in CONTRIBUTING.md